shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/TxBody.hs

```diff
@@ -365,7 +365,7 @@ instance Era era => FromCBOR (Wdrl era) where
 instance Era era => ToJSON (PoolParams era) where
   toJSON pp =
     Aeson.object
-      [ "publicKey" .= _poolPubKey pp,
+      [ "publicKey" .= _poolId pp, -- TODO publicKey is an unfortunate name, should be poolId
```

Comment:
I am assuming that I cannot make this change now, since it would break downstream code that depends on it.